cloud-inquisitor/aws_metadata.go

```diff
@@ -0,0 +1,11 @@
+package cloudinquisitor
+
+type AWSResourceMetadata struct {
+	AccountID       string `json:"AccountID" mapstructure:"AccountID"`
+	Region          string `json:"Region" mapstructure:"Region"`
+	ResourceID      string `json:"ResourceID" mapstructure:"ResourceID"`
+	ResourceAltID   string `json:"ResourceAltID" mapstructure:"ResourceAltID"`
```

Comment:
What is an Alt ID?

Reply:
Per how AWS design some of their products, they will have different identifiers (which varies by products, which is needed to pull resource data from API), one example would be RDS e.g.

it has ARN: arn:aws:rds:us-west-2:xxxxxx:db:db_name
it has an Resource id: db-ABCDEF1234567890
it also has DB instance id: cloud-inquisitor-new

The thing I am trying to resolve here is for Resource ID, it should use a naming standard which is universal per provider. For AWS it should be ARN, and for AltID, it should be sth which can be used directly to fetch resource from AWS API.

Reply:
Could we just used ResourceARN then?